packages/rolldown/package.json

```diff
@@ -126,7 +126,7 @@
     "why-is-node-running": "^3.0.0",
     "zod-to-json-schema": "^3.23.2"
   },
-  "optionalDependencies": {
+  "peerDependencies": {
```

Comment:
Why did you replace `optionalDependencies` to `peerDependencies`?

Reply:
This is a functional test, and I want it to support version replacement for `workspaces`. I have checked its code and it seems that there is some processing for `peerDependencies`, but it doesn't seem to work.